crates/swc_ecma_minifier/src/compress/optimize/inline.rs

```diff
@@ -209,6 +209,8 @@ impl Optimizer<'_> {
                                 )
                             }
 
+                            should_inline &= !usage.has_property_mutation;
```

Comment:
@Austaras We need a way to prevent the inline of variables initialized with arrays, I guess, but this seems to cause too much regression. Any ideas?

Reply:
Wait, wasn't this effectively the same as #7500?

Reply:
Oh... I'll check.